README.md

```diff
@@ -49,23 +182,22 @@ Tracer createWavefrontTracer(String application, String service) throws IOExcept
 
 Application tags determine the metadata (span tags) that are included with every span reported to Wavefront. These tags enable you to filter and query trace data in Wavefront.
 
-You encapsulate application tags in an `ApplicationTags` object.
+You encapsulate application tags in an `ApplicationTags` object, which is imported from the Wavefront Java SDK.
```

Comment:
Does this sound ok (technically)? 
I want to imply that we get it from the Wavefront Java SDK. Because afterward, we direct the users to the docs in the Wavefront Java SDK.